include/mbedtls/build_info.h

```diff
@@ -87,9 +87,10 @@
 #define MBEDTLS_MD_LIGHT
 #endif
 
-/* Auto-enable MBEDTLS_MD_LIGHT it one module needs it.
+/* Auto-enable MBEDTLS_MD_LIGHT if some module needs it.
```

Comment:
Very minor: perhaps "any" would be a better word?

Reply:
I've been reworking this comment in my working copy of https://github.com/Mbed-TLS/mbedtls/pull/7284